libbeat/dashboards/decode.go

```diff
@@ -36,21 +38,26 @@ var (
 )
 
 // DecodeExported decodes an exported dashboard
-func DecodeExported(result common.MapStr) common.MapStr {
+func DecodeExported(exported []byte) []byte {
 	// remove unsupported chars
-	objects := result["objects"].([]interface{})
-	for _, obj := range objects {
-		o := obj.(common.MapStr)
+	result := make([]byte, 0)
```

Comment:
Prefer `var result []byte` instead of allocating an empty slice. Happens in a few other places too.

https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices